catalyst/callbacks/checkpoint.py

```
@@ -495,6 +500,26 @@ def on_stage_start(self, runner: "IRunner") -> None:
         # map_location = {"cuda:%d" % 0: "cuda:%d" % rank}
         # ddp_model.load_state_dict(torch.load(CHECKPOINT_PATH, map_location=map_location))
 
+        if self.load_on_stage_start is not None:
+            need_full = False
+            file_exists = False
+            if isinstance(self.load_on_stage_start, str):
+                need_full = self.load_on_stage_start.endswith("full")
+                file_exists = os.path.isfile(
+                    os.path.join(self.logdir, f"{self.load_on_stage_start}.pth")
+                )
+            elif isinstance(self.load_on_stage_start, dict):
+                required_files = _get_required_files(self.logdir, self.load_on_stage_start).keys()
+                file_exists = all(os.path.isfile(file) for file in required_files)
+
+            if not self.load_on_stage_start is None and file_exists:
```

should we raise an error if nothing was found?